LayoutTests/media/video-played-collapse.html

```diff
@@ -71,6 +71,9 @@
             {
                 consoleWrite("<br><b><em>Test looping</em></b>");
 
+                // In GStreamer ports video.loop cannot reliably be set on a paused pipeline.
+                runSilently("video.play()");
```

Comment:
Why? Do we have a bug about this?

Reply:
From commit msg: This can be done only on a playing pipeline
though, because the segment seek is non-flushing. So when a seek is requested on a paused looping
player we have to enable flush, this is a corner case issue though.

Reply:
I can undo this change maybe, the seeks will flush but likely won't impact the test result, I don't remember why i did this change in the test actually, was almost 2 years ago :)

Reply:
Hm nope. CI confirms the test fails without this change.

Reply:
I think you can add the change and file a bug for that.